package.json

```diff
@@ -30,6 +30,7 @@
     "@react-aria/interactions": "3.5.0",
     "@react-spectrum/utils": "3.6.1",
     "@react-stately/toggle": "3.2.2",
+    "@reduxjs/toolkit": "^1.7.2",
```

Comment:
Try and use the `-E` flag when installing packages, so it pins to an exact version. This was a recommendation of a security firm (we have a CI job to lint this)